kratos/processes/reorder_and_optimize_modelpart_process.h

```diff
@@ -244,11 +244,11 @@ class ReorderAndOptimizeModelPartProcess : public Process
                     }
                 }
             }
-            
```

Comment:
you can take out the 
bool found
and the assignment,

other than that the loop is important since it is included in the outer loop.

i'll approve the branch since it is a hotfix which prevents compilation. I hope that the fix i propose fixes the warning, if it is not so, i'll open another branch to remove it ok?